lib/workers/repository/dependency-dashboard.ts

```diff
@@ -174,8 +174,11 @@ function appendRepoProblems(config: RenovateConfig, issueBody: string): string {
       'repository problems'
     );
     newIssueBody += '## Repository problems\n\n';
-    newIssueBody +=
-      'These problems occurred while renovating this repository.\n\n';
+    const repoProblemsHeader =
+      config.customizeDashboard?.['repoProblemsHeader'];
+    if (repoProblemsHeader) {
```

Comment:
can we use the old text in else block?

or should this text be optional?

@rarkins @secustor WDYT?

Reply:
I'm in favor for simply defaulting to empty string.

Reply:
i changed it, i agree it should be like that.